nnpdf_data/nnpdf_data/filter_utils/star/jet_dijet_2009_200gev.py

```diff
@@ -0,0 +1,319 @@
+"""This script provides the common filer to the jet and dijet STAR 2012 datasets.
+Files need to be parsed all together as there are correlations provided.
+"""
+
+import pathlib
+
+import numpy as np
+import pandas as pd
+import yaml
+
+from nnpdf_data.filter_utils.correlations import compute_covmat
+
+TOPOPLOGY_LIST = ["I", "SS", "OS", "A", "B", "C"]
+POL_UNC = 0.065
+LUMI_UNC = 0.0005
+HERE = pathlib.Path(__file__).parents[2]
+
+
+def read_1jet_data():
+    df3 = pd.DataFrame()
+    df4 = pd.DataFrame()
+
+    fnames = [
+        "../../new_commondata/STAR_2009_1JET_200GEV/rawdata/Table_3_ALL.csv",
+        "../../new_commondata/STAR_2009_1JET_200GEV/rawdata/Table_3_pT.csv",
+        "../../new_commondata/STAR_2009_1JET_200GEV/rawdata/Table_4_ALL.csv",
+        "../../new_commondata/STAR_2009_1JET_200GEV/rawdata/Table_4_pT.csv",
+        "../../new_commondata/STAR_2009_1JET_200GEV/rawdata/Table_5.csv",
+    ]
+
+    for fname in fnames:
+        if "3" in fname:
+            dfi = pd.read_csv(fname)
+            if "pT" in fname:
+                df3["pT"] = dfi["Parton Jet $p_T$ [GeV/c]"]
+
+            elif "ALL" in fname:
+                df3["ALL"] = dfi["$A_{LL}$"]
+                df3["stat_min"] = dfi["stat +"]
+                df3["stat_max"] = dfi["stat -"]
+                df3["sys_min"] = dfi["sys +"]
+                df3["sys_max"] = dfi["sys -"]
+
+            df3["abs_eta_min"] = 0.0
+            df3["abs_eta_max"] = 0.5
+            df3["abs_eta"] = (df3["abs_eta_min"] + df3["abs_eta_max"]) / 2
+            df3["sqrts"] = 200
+
+        elif "4" in fname:
+            dfi = pd.read_csv(fname)
+            if "pT" in fname:
+                df4["pT"] = dfi["Parton Jet $p_T$ [GeV/c]"]
+
+            if "ALL" in fname:
+                df4["ALL"] = dfi["$A_{LL}$"]
+                df4["stat_min"] = dfi["stat +"]
+                df4["stat_max"] = dfi["stat -"]
+                df4["sys_min"] = dfi["sys +"]
+                df4["sys_max"] = dfi["sys -"]
+
+            df4["abs_eta_min"] = 0.5
+            df4["abs_eta_max"] = 1.0
+            df4["sqrts"] = 200
+
+        elif "5" in fname:
+            dfc_col = pd.read_csv(fname)
+
+            dfc = pd.DataFrame()
+            biny = 1
+            for i in range(len(dfc_col)):
+                if dfc_col.loc[i, "binx"] == "-":
+                    biny = float(dfc_col.loc[i, "val"])
+                else:
+                    binx = float(dfc_col.loc[i, "binx"])
+                    dfc.loc[binx, biny] = dfc_col.loc[i, "val"]
+
+            dfc = dfc.astype("float")
+
+        else:
+            print("ERROR: Unknown table number detected! Check input files.")
+
+    df = pd.concat([df3, df4], ignore_index=True)
+    df["pol"] = POL_UNC * df["ALL"]
```

Comment:
I'd drop bot `pol` and `lumi` uncertainties as they are not mentioned in the tables/paper explicitly,
so they should be already included in the `sys`.